### ISSUE TITLE
GP minilab - workflow: do we need the physician to be able to see a previous version of a result ?

### ISSUE THREAD

**Issue by AlexisVZ-MDS** [hl7-be/lab#289]
Hello,

Currently we only show the latest version, so after importing the result of STEP2, the result of STAP1 is not accessible anymore. Is this OK ?

**Comment by bdc-ehealth** [comment]
@AlexisVZ-MDS ,

yes, that is OK. We see that the clinical biologists prefer that it is clearly indicated which value has changed. Showing the old value is not necessary.

**Comment by AlexisVZ-MDS** [comment]
@bdc-ehealth Thx ! But is indicating which values have changed an homologation criterion for the minilab ? 
If so, for me, that would be a new criterion ... To document in the Google Sheet with the planning of minilabs, so everyone is aware of it ?

**Comment by bdc-ehealth** [comment]
@AlexisVZ-MDS that will depend on the judgment of the clinical biologist. He/she determines if the result is medically clear and unambiguous enough. I can only say that they prefer to see if a value has changed.
